py/src/pathfinder_worker/call.py

```diff
@@ -879,12 +872,48 @@ def create_general_config(chain_id):
     )
 
     assert general_config.cairo_resource_fee_weights[f"{N_STEPS_RESOURCE}"] == 0.05
-    assert (
-        general_config.cairo_resource_fee_weights[f"{BITWISE_BUILTIN}_builtin"] == 12.8
-    )
```

Comment:
Any idea what these asserts are actually for?

Reply:
I guess to test that the weights got applied in the general config. I remember there were some issues with that  (not specifying the input dict in the right format).